torch/csrc/distributed/c10d/ProcessGroup.hpp

```diff
@@ -126,6 +131,13 @@ class TORCH_API ProcessGroup : public torch::CustomClassHolder {
     return backendType_;
   };
 
+  inline bool backendSupportsSequenceNumbers(BackendType backendType) {
+    if (backendType == BackendType::GLOO || backendType == BackendType::NCCL ||
+        backendType == BackendType::XCCL || backendType == BackendType::UCC)
```

Comment:
Do you make sure that we need to support this sequence number?

Reply:
Sequence number used by RECORD_PARAM_COMMS. so we need it